pages/case-characterisations.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <loading-message v-if="loading" />
+  <map-component
+    v-else
+    :pageInfo="pageInfo"
+    :originalMarkers="originalMarkers"
+    :filtersGeneralInfo="filtersGeneralInfo"
+    :filterProperties="filterProperties"
+    iconBehaviour="constant"
+  >
+  </map-component>
+</template>
+
+<script>
+import { SPARQLQueryDispatcher } from "~/assets/js/SPARQLQueryDispatcher";
+import APIDataHandler from "~/assets/js/APIDataHandler";
+import json from "../big-query-output.json";
+import MapComponent from "../components/MapComponent.vue";
+import LoadingMessage from "../components/LoadingMessage.vue";
+import Swal from 'sweetalert2';
+
+definePageMeta({
+   layout: 'default'
+ })
+
+export default {
+  components: { MapComponent, LoadingMessage },
+  data: () => ({
+    pageInfo: {
+      title: "Primary and Secondary Characterisations",
+      html: '<div>This map shows the geographical residence location for each accused witch in Scotland taken from the Survey of Scottish Witchcraft Database. Out of the <b class="font-bold">3212</b> accused witches whose names are known, the residence for <b class="font-bold">3142</b> witches has been located. The majority of the residences are accurately located down to the precise settlement, while others range from parish to county depending on the records surviving for each accused witch. There is a total of 821 different locations recorded in the database; all but 25 of these have been identified. The remaining unidentified place-names are currently recorded as \‘County of’\ on the map.</div>',
+      footer: "witches.is.ed.ac.uk",
+      confirmButtonText: "Close",
+      type: "info",
+      showCloseButton: true,
+    },
+    queryOutput: json,
+    sparqlUrl: "https://query.wikidata.org/sparql",
+    wikiPages: [],
+    loading: true,
+    originalMarkers: [],
+    filtersToFind: [
+      ["primary", "constant"],
+      ["secondary", "constant"],
+    ],
+    filtersGeneralInfo: {
+      title: "Case characterisations",
+      filtersShowing: true,
+    },
+    filterProperties: {
+      primary: {
+        label: "Primary",
+        description: "In examining, the documents of the 3,837 people who were accused of witchcraft in Scotland, the Survey of Scottish Witchcraft project team determined there were primary and secondary aspects of the cases that could be used to characterise them. For more information about the characteristics used below, please refer to the Survey’s glossary of terms here: <a href='https://witches.hca.ed.ac.uk/glossary/' target='_blank'>Survey Glossary</a>",
```

Comment:
The info icon here has no gap between the word Primary and the icon itself. We should make sure there is some whitespace between. I also think instead of an info icon, here we should use a _?_ icon as this is typical to explain something. This would work well as these are displayed as tooltips rather than the modals that the typical info popups are doing.

If you know of any other places we have similar tooltips not on this page, please could they be updated as well. Thanks Ruby

Reply:
have made screen readerable

Reply:
Thanks Ruby. For the wee popup, there is a little arrow that is currently pointing to the text, midway through. Is it possible to change this to point instead to the icon?

Also I'm not sure this is fully accessible yet. I'd be expecting to see an aria label and also some visually hidden text. [This blog](https://www.joshwcomeau.com/snippets/react-components/visually-hidden/) is for React but should give you an idea of what I mean

Reply:
Have added aria label and hidden text.

Was trying to move the arrow but was struggling with this as the Filter titles are different lengths the icon is in different places each time. Currently the tooltip also appears when you hover over the description not just the icon, was wondering if you though this was something that should be changed?

Reply:
Lets leave this for now. Too much work and we are under time pressure so I will resolve this issue and raise one to solve long term. Thanks Ruby